### ISSUE TITLE
Uppercase case in generator : no fields created in DB

### ISSUE THREAD

**Issue by hubyhuby** [markbates/pop#203]
Uppercase case for the name of the variable has an incidence on the creation in the DB.
This syntax works:
buffalo g resource test userid:uuid

Whereas this syntax doesn t create the apropriat columns in the pq database. And doesn t give a warning during the creation of the ressource
buffalo g resource test UserId:uuid

With this syntax the new test would fire something like :  "500 eror: pq: column...  does not exist"

Bottom line, whether my syntax is wrong, then there should be an error saying : "Syntax error: you can t do this"
Or It is a proper syntax and there is a bug in pop.

**Comment by hubyhuby** [comment]
I run in another similar syntax strange thing, if I enter fields with "_id" or "-id" the database doesn t seem to be generated either... The structs looks nice but when you want to do something from the web browser like New it says the fields doesn t exist in the DB...

**Comment by markbates** [comment]
Doing user_id:uuid works as expected, and is the “correct” way to do it. If that doesn’t work correctly for you, you’ll need to provide steps to reproduce, because I can’t. 

I haven’t tried the other ways, but I definitely agree if they’re not producing the correct results, it’s definitely bug.

**Comment by markbates** [comment]
Also what is a MAJ?

**Comment by markbates** [comment]
Just to through some more substance on this.

```
$ soda g model widget userId:uuid FooID:uuid bar_id:uuid bazid:uuid
```

produces

```go
type Widget struct {
	ID        uuid.UUID `json:"id" db:"id"`
	CreatedAt time.Time `json:"created_at" db:"created_at"`
	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
	UserId    uuid.UUID `json:"userId" db:"userId"`
	FooID     uuid.UUID `json:"FooID" db:"FooID"`
	BarId     uuid.UUID `json:"bar_id" db:"bar_id"`
	Bazid     uuid.UUID `json:"bazid" db:"bazid"`
}
```

```
create_table("widgets", func(t) {
	t.Column("id", "uuid", {"primary": true})
	t.Column("user_id", "uuid", {})
	t.Column("foo_id", "uuid", {})
	t.Column("bar_id", "uuid", {})
	t.Column("bazid", "uuid", {})
})
```

only `bar` and `baz` produced struct tags that match the column names

**Comment by hubyhuby** [comment]
Hi Mark,
MAJ means Upper case in French :D Sorry for mixing languages .I am glade you found a solution.
I just put this for reference : 
If I type "rank:nulls.string"  with a "s" at string
I get when launching the app  "models/product.go:43: cannot refer to unexported name nulls.string"*
If I do 
"rank:nulls.String" it all goes well ("S")

----------------------------------
I understand that for the generator "Any other type passed it will be be passed straight through to the underlying database." Which is nice. Maybe a warning saying "those types are not recognised in standard by buffalo"
